OD-326 [Fix] Grid components will be sized correctly.

YaroslavOvdii opened the pull request:
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-326

## Description
This issue occurred because of this [part](https://github.com/Fliplet/fliplet-widget-metro/blob/master/vendor/freewall.js#L669).
Where `maxX` was zero (0). To ensure that element doesn't get a 0 width we use Math.max to get the biggest value.

## Screenshots/screencasts
https://share.getcloudapp.com/o0uq21mX

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@AndrRyaz @ivandevupp

YaroslavOvdii committed:
fix for small size items

tonytlwu reviewed (changes requested):
Issue reference https://weboo.atlassian.net/browse/OD-310 is incorrect.

And also, we avoid modifying vendor libraries where possible. Please try other solutions or show that you have tried other solutions. If the library works for everyone else who has built websites using the library, then there should be no need to update the library.

YaroslavOvdii commented:
@tonytlwu 
I've tried another way to fix that issue.
In order to do so, I've looked into the [metroInit.js](https://github.com/Fliplet/fliplet-widget-metro/blob/master/js/metroInit.js) where we init the `freewall` library.
And tried to change the size of the elements at this [point](https://github.com/Fliplet/fliplet-widget-metro/blob/master/js/metroInit.js#L64). But I've discovered that at this point in the code width value of the element is correct.
And after this method is complete we are calling the inner [method ](https://github.com/Fliplet/fliplet-widget-metro/blob/master/vendor/freewall.js#L576)of the library. And in it, we are trying to put our element into the layout matrix where it fits. 
[Previously](https://github.com/Fliplet/fliplet-widget-metro/pull/28#discussion_r510207981) we made adjustments to [this code](https://github.com/Fliplet/fliplet-widget-metro/blame/master/vendor/freewall.js#L668) so it would behave as expected.
Therefore the original [library](http://kombai.github.io/freewall/) worked like [this](https://github.com/Fliplet/fliplet-widget-metro/pull/28#discussion_r510207981) we corrected that. Anfotchunatly I couldn't find in the documentation of the lib how we can do it with its API so I've changed the behavior by altering the lib itself.

So I still think that changing that I've made is the best solution so far.
1. Library that we using is hosted together with the component so it won't change accidentally with updating npm packages.
2. We are using the last version of library 1.05, and the last update of the git repository was 5 years ago.

tonytlwu commented:
@YaroslavOvdii Thanks for the thorough explanation. It sounds like you ended up correcting a patch you previously applied. Since this is already a patch we had before, I'm OK with the vendor library being updated.

tonytlwu reviewed (approved):
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: Grid component users
Verify: Grid items are correctly sized on all device sizes

Squash and merge: Ensures grid items are sized correctly
